mtbf_driver/tests/mtbf/keyboard/test_mtbf_keyboard.py

```diff
@@ -4,7 +4,8 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 from mtbf_driver.MtbfTestCase import GaiaMtbfTestCase
-from gaiatest.apps.contacts.app import Contacts
+#from gaiatest.apps.contacts.app import Contacts
```

Comment:
Suggesting to remove commented code, it will leave in previous revision though.  It's ok if you feel like to keep.